spec/lib/tmuxinator/config_spec.rb

```diff
@@ -329,6 +329,37 @@
   describe "#validate" do
     let(:default) { Tmuxinator::Config::LOCAL_DEFAULT }
 
+    context "when a project config file is provided" do
```

Comment:
If we're going to keep the existing implementation (`raise`ing errors in the validations) we should test for that behavior.